back/spec/acceptance/phases_spec.rb

```diff
@@ -432,5 +432,31 @@
         expect(json_response).to eq({ data: { totalSubmissions: 3 } })
       end
     end
+
+    delete 'web_api/v1/phases/:id/inputs' do
+      let(:project) { create(:project_with_active_native_survey_phase) }
+      let(:active_phase) { project.phases.order(:start_at).last }
+      let(:id) { active_phase.id }
+
+      example 'Delete all inputs of a phase' do
+        ideation_phase = create(
+          :phase,
+          project: project,
+          participation_method: 'ideation',
+          start_at: (Time.now - 2.months),
+          end_at: (Time.now - 1.month)
+        )
+        create_list :idea, 2, project: project, phases: [active_phase]
+        create :idea, project: project, phases: [ideation_phase]
+
+        do_request
+        expect(response_status).to eq 200
+        expect(Phase.find(id)).to eq active_phase
+        expect(project.reload.ideas_count).to eq 1
+        expect(active_phase.reload.ideas_count).to eq 0
+        expect(ideation_phase.reload.ideas_count).to eq 1
+        expect(Idea.count).to eq 1
+      end
+    end
```

Comment:
Please add a spec that describes what happens when deleting is not allowed, otherwise we do not have proof that the policy class is used correctly.